package.json

```
@@ -59,10 +59,34 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
     "babel-core": "^7.0.0-bridge.0",
+    "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-formatter-summary": "^1.0.4",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "husky": "^3.0.9",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.1.0",
-    "parcel-bundler": "^1.12.3",
-    "prettier": "^1.16.4"
+    "lint-staged": "^9.4.2",
+    "parcel-bundler": "^1.12.4",
+    "prettier": "^1.18.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx}": [
+      "eslint --fix",
+      "prettier —write",
```

Should be double `en dash` instead of a single `em dash` --> https://katherinewikoff.files.wordpress.com/2013/01/hyphen-vs-dash.jpg
```suggestion
      "prettier --write",
```

---

Oops, sorry about that! might have added this before merging! It's fixed now.

---

@theajr Right now these are two different dashes. I hope it works properly ;)

---

@jacekk It is working fine!